static/src/stylesheets/_head.amp-common.scss

```diff
@@ -20,6 +20,7 @@
 @import 'amp/_content';
 @import 'amp/_social';
 @import 'amp/_onward-tones';
+@import 'amp/_article-tones';
```

Comment:
If we add this here, can we remove from https://github.com/guardian/frontend/blob/master/static/src/stylesheets/head.amp.scss#L3 ?

There are also liveblog specific tones: https://github.com/guardian/frontend/blob/master/static/src/stylesheets/head.amp-liveblog.scss#L4